sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/APISpec.groovy

```diff
@@ -339,6 +346,28 @@ class APISpec extends StorageSpec {
         }
     }
 
+    static BlobServiceClient getOauthBlobServiceClient() {
+        def builder = new BlobServiceClientBuilder()
+            .endpoint("https://" + env.primaryAccount.name + ".blob.core.windows.net")
```

Comment:
please don't do that. `env.primaryAccount.blobEndpoint` will do it for you. Not hardcoding this format is important when we get to testing non public clouds.